src/app/mode-switcher/mode-switcher.component.ts

```diff
@@ -3,16 +3,13 @@ import { CUSTOM_ELEMENTS_SCHEMA, Component, EventEmitter, Output } from '@angula
 @Component({
   selector: 'app-mode-switcher',
   templateUrl: './mode-switcher.component.html',
-  styleUrls: ['./mode-switcher.component.css'],
   standalone: true,
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
 })
 export class ModeSwitcherComponent {
-
   @Output() modeToggle = new EventEmitter<void>();
-  
-  toggleMode(){
-    this.modeToggle.emit()
-    console.log('hej')
+
+  toggleMode() {
+    this.modeToggle.emit();
```

Comment:
I think the @Output declaration/instantiation & the `this.modeToggle.emit();` can be removed since it doesn't do anything

Reply:
It is actually used in app component? to switch to dark mode.

Reply:
that's correct, my bad

Reply:
Thank you both for your comments and checking it thou ;)